notebooks/Fine_tuning_Code_LLM_on_single_GPU.ipynb

```diff
@@ -0,0 +1,5363 @@
+{
```

Comment:
<p>Do we need to provide links to the models mentioned?</p><p> </p><p>Is it also worth mentioning that a meta goal of this notebook is to be able to run the entire process in a single notebook and single GPU (IIUC)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>These are links, they just don't render weirdly in the ReviewNB app, but if you hover, you can see that they are there :) But I'll add a note on the meta goal of the notebook. </p>